### DISCUSSION TITLE
Not getting more than 9 plots running parallel

### DISCUSSION THREAD

**Discussion by kianschki** [swar/Swar-Chia-Plot-Manager#807] [General]
Hi all,

this is my setting and I have restarted the manager several times:
  - name: micron
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: /media/harry/Spitzhacke
    temporary2_directory:
    destination_directory: [/media/harry/A1_8/single,/media/harry/A2_8/single,/media/harry/A3_10/single,/media/harry/A4_10/single]
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 12
    max_concurrent_with_start_early: 13
    initial_delay_minutes: 0
    stagger_minutes: 30
    max_for_phase_1: 6
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 300
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10

But I'm not getting more than 9 run in parallel and not more than 3 run in phase 1. I have no idea why :(

**Comment by Lyushen** [comment]
What is your global value such as
global: - category
max_concurent:? 
It should be 9

**Comment by kianschki** [comment]
Thanks a lot! I think this was the problem, I overlooked this settings!
